easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.1.2'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = "TopHat is a fast splice junction mapper for RNA-Seq reads."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
I suspect this software has any BLAS/LAPACK, so we can drop this down to the `gompi` level instead.

Reply:
This isn't resolved? Or does it have a dependency on BLAS/LAPACK that i missed?

Reply:
@hcartiaux what @Micket is suggesting is to change this line to:
```
toolchain = {'name': 'gompi', 'version': '2019b'}
```